includes/definitions/discovery/routeros.yaml

```diff
@@ -151,17 +178,24 @@ modules:
                 -
                     oid: mtxrHlPower
                     num_oid: .1.3.6.1.4.1.14988.1.1.3.12.
-                    descr: 'System'
-                    index: 'mtxrHlPower.{{ $index }}'
+                    divisor: 10
+                    descr: 'Power Usage'
+                    index: '1'
         fanspeed:
             data:
                 -
                     oid: mtxrHlFanSpeed1
                     num_oid: .1.3.6.1.4.1.14988.1.1.3.17.
                     descr: 'Fan #1'
-                    index: 'mtxrHlFanSpeed1.{{ $index }}'
+                    index: '0'
                 -
                     oid: mtxrHlFanSpeed2
                     num_oid: .1.3.6.1.4.1.14988.1.1.3.18.
                     descr: 'Fan #2'
-                    index: 'mtxrHlFanSpeed2.{{ $index }}'
+                    index: '1'
+        signal:
+            data:
+                -
+                    oid: mtxrWlRtabStrength
```

Comment:
This doesn't seem right.  It seems to be wireless client signal strength...  It is indexed by MAC, which caused several issues.

It is also similar to the MSA issue with the string based index.

I would suggest dropping this sensor.

Reply:
And don't forget to update the test data.

Reply:
Updated back to php